### DISCUSSION TITLE
Custom DNS config

### DISCUSSION THREAD

**Discussion by ibrahimk157** [angristan/wireguard-install#417] [Help]
Hi,

I wanted to suggest adding a custom dns config to the script such that it would ask us for our preferred DNS servers. Since I want to use the AdGuard DNS through it.

thanks

**Comment by houmie** [comment]
Did you actually try the script? It is using AdGuard by default.

**Comment by ibrahimk157** [comment]
> Did you actually try the script? It is using AdGuard by default.

Yes of course I’ve tried it. However, the ads still slip through which wasn’t the case back when I’ve used OpenVPN with the AdGuard DNS. So not sure what’s going on.
